airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/errors/messages/OracleErrorMessage.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.base.errors.messages;
+
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_HOST_OR_PORT;
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_USERNAME_ACCOUNT_IS_LOCKED;
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_USERNAME_OR_PASSWORD_OR_DATABASE_OR_USER_ACCESS_DENIED;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.ORACLE;
+
+import io.airbyte.integrations.base.errors.utils.ConnectorName;
+
+public class OracleErrorMessage extends ErrorMessage {
+
+  {
+    CONSTANTS.put("99999", INCORRECT_USERNAME_ACCOUNT_IS_LOCKED);
```

Comment:
Similar to my comment on `MySQL error SQLSTATES` it seems that these mappings don't quite match the ones defines in the [Oracle SQLSTATE docs](https://docs.oracle.com/cd/E18283_01/appdev.112/e10827/appd.htm) where `08006` is actually defined to be "connection failure" and actually was hopeful it would solve [this ticket](https://github.com/airbytehq/airbyte/issues/12962) that I'm currently working on

Similarly `99999` is said to be a catch all error code so don't think we can say the account is locked unless I'm missing some crucial information